### ISSUE TITLE
Management of CSV files

### ISSUE THREAD

**Issue by dkids** [DobyTang/LazyLibrarian#1390]
I'm using a shell script to convert something like https://www.goodreads.com/list/show/73737.YA_Novels_of_2017 into a csv file.

I would like for LL to read the file, then delete it when done.   That would mean having separate directories for importing and exporting csv files, which is reasonable to me.

**Comment by philborman** [comment]
We don't delete it, but if you process it again nothing will happen, we already imported it.  We won't produce duplicate entries. I could fairly easily add a switch to delete the csv file after import completes, but wouldn't need separate directories, what's the advantage?

Also, the link you gave is a goodreads Listopia feed, which we can import directly, no need for csv conversion. Just put the link in as an rss feed in lazylibrarian config providers tab and we will import the list, try to import all books in the list (but not all books by all authors!), and then periodically monitor the list for any new additions.  It's a __very__ long list though with 100 books per page and at least 11 pages, so you can limit the number of pages by setting the Pages option, 1 for first 100 books only, 2 for 200...
The Pages option is labelled Priority for most rss feeds, but changes to Pages for wishlists/listopia. Not sure if the label gets changed instantly, might need a refresh.

**Comment by dkids** [comment]
OK, that's good to know about CSV files.   I had looked for activity under jobs but it's not there.   When I pushed the "Import CSV" button it wasn't clear what was going on except that I would get a message about 1 file being imported.   Just now I got "IMPORTCSV already running".   I can see IMPORTCSV in the log files but it's hard to understand what's happening, especially with being at debug level 3 right now.

That's an interesting idea about using the RSS feed on an ongoing basis.   Some of the lists are active, with hundreds of voters, but most only have a few.   

I'm planning on submitting some additions to the Wiki so that things are documented for other people to reference.
